src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -4,6 +4,7 @@
 import carpet.fakes.ItemEntityInterface;
 import carpet.fakes.MobEntityInterface;
 import carpet.helpers.Tracer;
+import carpet.helpers.EntityPlayerActionPack;
```

Comment:
why that?

Reply:
umm, i forgot to remove it, cos I was using e.EntityPlayerActionPack.Jump()

Reply:
done